harness/determined/pytorch/_lr_scheduler.py

```diff
@@ -49,7 +51,15 @@ def __init__(
                 2. ``STEP_EVERY_BATCH``: Determined will call scheduler.step() after every
                    ``frequency`` training batch(es). No arguments will be passed to step().
 
-                3. ``MANUAL_STEP``: Determined will not call scheduler.step() at all.
+                3. ``STEP_EVERY_OPTIMIZER_STEP``: Determined will call scheduler.step() in sync
```

Comment:
Can we alter the docs for ``STEP_EVERY_BATCH`` to suggest that ``STEP_EVERY_OPTIMIZER_STEP`` should be preferred?

Reply:
sure, makes sense.